daemon/logic/worklog.go

```diff
@@ -70,6 +72,21 @@ func (w *Worklog) List(ctx context.Context, orgID *identity.ID) ([]apitypes.Work
 		items = append(items, item)
 	}
 
+	keypairs, err := w.engine.client.KeyPairs.List(ctx, org.ID)
+	if err != nil {
+		return nil, err
+	}
+
+	if len(keypairs) == 0 {
```

Comment:
What about revoked keypairs? technically those are missing as well -- if a user has no keys for an org.